### DISCUSSION TITLE
NABox 4.x and Harvest

### DISCUSSION THREAD

**Discussion by statdigger** [NetApp/harvest#2996] [General]
Hi all,

has anyone tried the new NABox 4.x with Harvest, if so have you had any issues?.
Do we still use the "..linux_amd64.tar.gz" release of Harvest with it?
Does it still use both ZAPI & REST collectors?

NABox 4 OVA now ships with Flatcar Container Linux and uses Victoria Metrics whereas NABox 3 used to ship with Alpine Linux and used Prometheus.

I only retain 2 months of metrics so I won't need to migrate stats from the Prometheus database.

Thanks for any pointers you can provide.

**Comment by cgrinds** [comment]
hi @statdigger lots of folks have mentioned that they have upgraded on Discord. You'll probably get more answers there if you can join. https://github.com/NetApp/harvest/blob/main/SUPPORT.md#discord

Yes, the `tar.gz` is still used.
Yes, NAbox has not changed anything about Harvest and Harvest still supports ZAPI, REST, StorageGRID, and EMS collectors.

**Reply by statdigger** [reply]
Many thanks for the prompt reply and the information cgrinds. I'll take a look on discord too.
